src-built-in/components/myApps/src/stores/StoreActions.js

```diff
@@ -174,9 +174,15 @@ function loadInstalledConfigComponents(cb = Function.prototype) {
 		componentNameList.map(componentName => {
 			// If the app is already in our list move on
 			if (appInAppList(componentName)) return;
-			let component = componentList[componentName];
+			const component = componentList[componentName];
+			let launchableByUser;
+			try {
+				launchableByUser = component.foreign.components["App Launcher"].launchableByUser;
```

Comment:
Instead of using `try/catch`, you can use `lodash.get`.  I think it is a little cleaner.  See an example in some code I wrote here: https://github.com/ChartIQ/finsemble/pull/1435/files#diff-b0f6bb271af0bec4b31c620511df3755R249

Reply:
I would concur with changing this to lodash's get currently. Ian is assessing options to address the need of this, including possibly using the [optional chaining](https://github.com/tc39/proposal-optional-chaining) that will be built into JavaScript soon. So having one easy path of things to look for to update if that move happens would be nicer than needing to go filter between lodash get usage and any possibly try/catches that exist only for this purpose.